does not install with Zeek v6.0.0

Installing:

```
$ zkg -vvv install zeek-plugin-tds
2023-09-06 13:29:00 DEBUG    init Manager version 2.14.0
2023-09-06 13:29:00 DEBUG    found source clone of "zeek" at /opt/zeek/var/lib/zkg/clones/source/zeek
2023-09-06 13:29:00 DEBUG    getting info on "zeek-plugin-tds"
2023-09-06 13:29:01 DEBUG    checked out "zeek/amzn/zeek-plugin-tds", branch/version "1.1.0"
2023-09-06 13:29:01 DEBUG    getting info on "zeek/amzn/zeek-plugin-tds"
2023-09-06 13:29:02 DEBUG    checked out "zeek/amzn/zeek-plugin-tds", branch/version "1.1.0"
The following packages will be INSTALLED:
  zeek/amzn/zeek-plugin-tds (1.1.0)

Proceed? [Y/n] y
2023-09-06 13:29:04 INFO     Skipping unit tests for "zeek/amzn/zeek-plugin-tds": no test_command in metadata
2023-09-06 13:29:04 DEBUG    installing "zeek/amzn/zeek-plugin-tds"
Installing "zeek/amzn/zeek-plugin-tds"2023-09-06 13:29:05 DEBUG    staging "zeek/amzn/zeek-plugin-tds": version 1.1.0
2023-09-06 13:29:05 DEBUG    building "zeek/amzn/zeek-plugin-tds": running build_command: ./configure && make
2023-09-06 13:29:05 INFO     installing "zeek/amzn/zeek-plugin-tds": writing build log: /opt/zeek/var/lib/zkg/logs/zeek-plugin-tds-build.log
.
Failed installing "zeek/amzn/zeek-plugin-tds": package build_command failed, see log in /opt/zeek/var/lib/zkg/logs/zeek-plugin-tds-build.log
error: incomplete installation, the follow packages failed to be installed:
  zeek/amzn/zeek-plugin-tds (1.1.0)
```

Looking at the build.log:

```
=== STDERR ===
Usage: zeek-config [OPTIONS]

Basic options:

  --build_type          Zeek build type as per cmake, lower case (e.g. 'relwithdebinfo')
  --prefix              Toplevel Zeek distribution installation directory
  --version             Zeek version number
  --zeek_dist           Toplevel directory of source tree the distribution built from
  --zeekpath            ZEEKPATH environment variable paths for this distribution

Specific directories in the Zeek distribution:

  --btest_tools_dir     Zeek-related BTest tooling
  --cmake_dir           Zeek's cmake modules
  --config_dir          Configuration files for cluster topology, zkg, etc
  --include_dir         C/C++ header folders for Zeek and related components, colon-separated
  --lib_dir             Toplevel folder for shared libraries, Python packages, etc
  --plugin_dir          Native-code Zeek plugins
  --python_dir          Python packages (Broker, ZeekControl, zkg, etc)
  --script_dir          Toplevel folder for Zeek scripts
  --site_dir            Site-specific Zeek scripts

Toplevel installation directories for third-party components:

  --binpac_root         BinPAC compiler
  --broker_root         Broker communication framework

Feature tests:

  --have-spicy-analyzers  Prints 'yes' if built-in Spicy analyzers are available; exit code reflects result

CMake Warning at /opt/zeek/share/zeek/cmake/ZeekPlugin.cmake:141 (message):
  Package requires CMake 3.0 which is less than Zeek's requirement (3.15.0).
  This will likely cause build failures and should be fixed.
Call Stack (most recent call first):
  CMakeLists.txt:6 (include)


CMake Warning (dev) at /opt/zeek/share/zeek/cmake/ZeekPlugin.cmake:120 (if):
  Policy CMP0057 is not set: Support new IN_LIST if() operator.  Run "cmake
  --help-policy CMP0057" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  IN_LIST will be interpreted as an operator when the policy is set to NEW.
  Since the policy is not set the OLD behavior will be used.
Call Stack (most recent call first):
  /opt/zeek/share/zeek/cmake/ZeekPluginDynamic.cmake:77 (zeek_next_pac_block)
  /opt/zeek/share/zeek/cmake/ZeekPlugin.cmake:196 (zeek_add_dynamic_plugin)
  /opt/zeek/share/zeek/cmake/ZeekPluginCommon.cmake:68 (zeek_add_plugin)
  CMakeLists.txt:13 (zeek_plugin_end)
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Error at /opt/zeek/share/zeek/cmake/ZeekPlugin.cmake:120 (if):
  if given arguments:

    "arg" "IN_LIST" "separators"

  Unknown arguments specified
Call Stack (most recent call first):
  /opt/zeek/share/zeek/cmake/ZeekPluginDynamic.cmake:77 (zeek_next_pac_block)
  /opt/zeek/share/zeek/cmake/ZeekPlugin.cmake:196 (zeek_add_dynamic_plugin)
  /opt/zeek/share/zeek/cmake/ZeekPluginCommon.cmake:68 (zeek_add_plugin)
  CMakeLists.txt:13 (zeek_plugin_end)


=== STDOUT ===
Build Directory        : build
Zeek Source Directory   : 
-- The C compiler identification is GNU 12.2.0
-- The CXX compiler identification is GNU 12.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Found OpenSSL: /usr/lib/x86_64-linux-gnu/libcrypto.so (found version "3.0.9")  
-- Found BinPAC: /opt/zeek/bin/binpac  
-- Found BifCl at /opt/zeek/bin/bifcl
-- Setting plugin CMAKE_BUILD_TYPE to RelWithDebInfo
-- Configuring incomplete, errors occurred!
See also "/opt/zeek/var/lib/zkg/clones/package/zeek-plugin-tds/build/CMakeFiles/CMakeOutput.log".
```

I know the CAF stuff is deprecated and should be removed. I'm testing a fix now (setting minimum cmake 3.15.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,5 @@
 
-cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.15 FATAL_ERROR)
 
 project(Plugin)
 

--- a/configure
+++ b/configure
@@ -25,7 +25,6 @@ Usage: $0 [OPTIONS]
     --install-root=DIR         Path where to install plugin into
     --with-binpac=DIR          Path to BinPAC installation root
     --with-broker=DIR          Path to Broker installation root
-    --with-caf=DIR             Path to CAF installation root
     --with-bifcl=PATH          Path to bifcl executable
     --enable-debug             Compile in debugging mode
 EOF
@@ -85,11 +84,6 @@ while [ $# -ne 0 ]; do
         --with-broker=*)
             append_cache_entry BROKER_ROOT_DIR PATH $optarg
             broker_root=$optarg
-            ;;
-
-        --with-caf=*)
-            append_cache_entry CAF_ROOT_DIR PATH $optarg
-            caf_root=$optarg
             ;;
 
         --with-bifcl=*)
@@ -154,9 +148,6 @@ if [ -z "$zeekdist" ]; then
         append_cache_entry BROKER_ROOT_DIR PATH `${zeek_config} --broker_root`
     fi
 
-    if [ -z "$caf_root" ]; then
-        append_cache_entry CAF_ROOT_DIR PATH `${zeek_config} --caf_root`
-    fi
 else
     if [ ! -e "$zeekdist/zeek-path-dev.in" ]; then
         echo "$zeekdist does not appear to be a valid Zeek source tree."

